index.html

```diff
@@ -1,259 +1,241 @@
+                    </button>
+                    <a class="navbar-brand page-scroll" href="#page-top">
+                        <img class="white-icon" src="css/img/icon-w.png" height="38" width="38">
+                        <img class="green-icon" src="css/img/icon.png" height="38" width="38"> 
+                        <div class="logofontposition">
+                            <span class="light">THE<br> SQUEEZY</span>LEMON
+                        </a>
                     </div>
                 </div>
-                   <div class="row headbtn">
-                <a href="#about" class="btn btn-circle page-scroll">
-                <i class="fa fa-angle-down animated"></i>
-                </a>
-            </div>
-            </div>
-        </div>
 
-
-    </header>
-
-
- 
-    <!-- Team -->
-    <section id="about">
-    <div class="about-body">
-        <div class="container content-section">
-            <div class="row row-centered">
-                <div class="col-md-6 marginhead col-centered">
-                    <h2>Meet the team</h2>
-                    <p>We're a strong team of two designers. We work remotely, all over the world. So we're in a lot of different environments that inspire us. This inspiration we can use for designing your brand.</p>
-                </div> 
-            </div>
-            <div class="row row-centered">
-                <div class="col-md-4 col-sm-6 col-centered"> 
-                    <div class="bouke">
-                        <img src="css/img/bouke.png" height="150" width="150">
-                        <h3>Bouke van Gestel</h3>
-                        <p> She's got a degree in Art, Communication & Design and Fine Arts. She can help you with branding, web design and illustrations. </p>
-                    </div>
-                </div>
-                <div class="col-md-4 col-sm-6 col-centered">
-                    <div class="nils">
-                        <img src="css/img/nils.png" height="150" width="150">
-                        <h3> Nils van Breen</h3/>
-                        <p> He's got a degree in Industrial Product Design, specialised in packaging. He can help you with branding design, packaging design and 3D modeling. </p>
-                    </div>
+                <!-- Collect the nav links, forms, and other content for toggling -->
+                <div class="collapse navbar-collapse navbar-right navbar-main-collapse">
+                    <ul class="nav navbar-nav">
+                        <!-- Hidden li included to remove active class from about link when scrolled up past about section -->
+                        <li class="hidden">
+                            <a href="#page-top"></a>
+                        </li>
+                        <li>
+                            <a class="page-scroll" href="#about">Team</a>
+                        </li>
+                        <li>
+                            <a class="page-scroll" href="#download">Services</a>
```

Comment:
I would change that to `href="#services"` and of course apply all the changes in the HTML and CSS that go with it.